gateway/gateway.go

```diff
@@ -533,7 +531,7 @@ func (g *gateway) DoCommand(ctx context.Context, cmd map[string]interface{}) (ma
 			g.readingsMu.Unlock()
 		}
 	}
-	if payload, ok := cmd[sendDownlinkKey]; ok {
+	if payload, ok := cmd[node.GatewaySendDownlinkKey]; ok {
```

Comment:
I don't love that the node has a SendDownlink variable - but I think we should just have a top level lorawan package containing all the functionality of node and gateway when you refactor this module.

Reply:
yeah I needed to have a key that both `gateway` and `node` can read, but gateway already imports node. once we get to refactoring we can maybe use an eng focus to map out the package structure and what we want implementers to have to implement.